zerver/models.py

```diff
@@ -1011,6 +1011,11 @@ def get_context_for_message(message):
 
 post_save.connect(flush_message, sender=Message)
 
+class Reaction(ModelReprMixin, models.Model):
+    sender = models.ForeignKey(UserProfile) # type: UserProfile
```

Comment:
I think `sender` is probably a confusing name for this; maybe just call it `user_profile`?  I think most people think about is as user who reacted, not a person who sent a reaction.